bindings/py/cpp_src/bindings/algorithms/py_TemporalMemory.cpp

```diff
@@ -352,6 +352,14 @@ Argument cell
 
 Returns the created segment (index handle).)");
 
+        py_HTM.def("cellsToColumns", &HTM_t::cellsToColumns,
+R"(Converts cells SDR to corresponding columns SDR.
```

Comment:
please test that this method works in py, I've had problems with this incomplete signature and I needed to specify the `py::arg("bla")` argument. See other methods in this file as example.

Reply:
it works for me when i imported and used the binding in python. But you are right, i wrote the arguments differently than the other methods.. pybind11 is quite unknown land for me.
Now it should be OK